Jenkinsfile

```
@@ -83,17 +78,10 @@ node('docker&&linux') {
                     sh 'wget --debug -O /dev/null --retry-connrefused --timeout 120 --tries=15 http://localhost:8080/versions'
                 }
             }
-
-            stage('Tag container as latest') {
-                if (pushToDocker) {
-                    echo "Tagging ${dockerImage} as latest"
-                    infra.withDockerCredentials {
-                        container.push('latest')
-                    }
-                }
-            }
         }
-
+        stage('Build and publish Docker image') {
+            buildDockerAndPublishImage('plugin-site-api', [unstash: 'build', targetplatforms: 'linux/amd64'])
```

```suggestion
            buildDockerAndPublishImage('plugin-site-api', [unstash: 'build', enablePublication: infra.isInfra(),targetplatforms: 'linux/amd64'])
```
Publish only on infra.ci.jenkins.io by setting `enabledPublication` to false if not on infra.
Needs https://github.com/jenkins-infra/pipeline-library/pull/784